Doc/reference/executionmodel.rst

```diff
@@ -232,13 +232,17 @@ Annotation scopes differ from function scopes in the following ways:
 .. versionadded:: 3.12
    Annotation scopes were introduced in Python 3.12 as part of :pep:`695`.
 
+.. versionchanged:: 3.13
+   Annotation scopes are now also used for type parameter defaults, as
```

Comment:
nit: the "now" makes it sound like annotation scopes previously weren't used for this
```suggestion
   Annotation scopes are also used for type parameter defaults, as
```